mlos_bench/mlos_bench/tests/optimizers/toy_optimization_loop_test.py

```diff
@@ -95,12 +95,12 @@ def test_mock_optimization_loop_no_defaults(
 def test_flaml_optimization_loop(mock_env_no_noise: MockEnv, flaml_opt: MlosCoreOptimizer) -> None:
     """Toy optimization loop with mock environment and FLAML optimizer."""
     (score, tunables) = _optimize(mock_env_no_noise, flaml_opt)
-    assert score == pytest.approx(60.15, 0.01)
+    assert score == pytest.approx(63.78, 0.01)
```

Comment:
Is this a result of the change reported in #839 or the change in the quantization?

Reply:
no, this is a FLAML issue. I'll happily revert this change once we merge in the FLAML version fix

Reply:
done. I've rolled back that change. the unit test should pass once we merge in the FLAML version fix